company_website/templates/base.haml

```diff
@@ -6,12 +6,12 @@
 {% load google_analytics_tags %}
 {% load pipeline %}
 {% stylesheet 'common' %}
-.google-analytics
-    %img.google-analytics-image{:src => "{% google_analytics %}", :alt => "Google Analytics"}
-
 {% block extra_head %}
 {% endblock %}
 
+.google-analytics
+    %img.google-analytics-image{:src => "{% google_analytics %}", :alt => "Google Analytics"}
+
```

Comment:
Why you moved this?

Reply:
**.google_analytics** element was causing extra head content to be placed in the body.
I can't get this to work otherwise: **haml** doesn't allow div to be placed in head, at least with current structure. Should we use **%head** tag explicitly, if at all possible? But even then I can't tell it will work, because when I place it in the **extra_head** block, it just vanishes altogether. 😕

Reply:
I just checked and even the explicit use of **%head** and **%body** tags doesn't help: the div is forced into the body anyway!

It is ill-advised to put divs in the head anyway, as it should be reserved for a meta-like data only. It has been documented by W3C for example.

Looks like everything is working against putting those elements in the head. 😛